.github/workflows/build_mpv_binary_and_deb_package.yml

```diff
@@ -0,0 +1,139 @@
+name: Build mpv binary and create debian package
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y \
+            build-essential \
+            clang \
+            yasm \
+            git \
+            meson \
+            ninja-build \
+            autoconf \
+            automake \
+            libtool \
+            pkg-config \
+            libfreetype6-dev \
+            libfribidi-dev \
+            libfontconfig1-dev \
+            libasound2-dev \
+            pulseaudio \
+            libx11-dev \
+            libxss-dev \
+            libvdpau-dev \
+            libgl-dev \
+            libxv-dev \
+            libjpeg-dev \
+            libssl-dev \
+            libx264-dev \
+            libmp3lame-dev \
+            libfdk-aac-dev \
+            python3-pip \
+            python3.12-venv \
+            libharfbuzz-dev \
+            nasm \
+            liblua5.1-0-dev \
+            liblua5.2-dev \
+            libxext-dev \
+            libxrandr-dev \
+            libxinerama-dev \
+            libxv-dev \
+            libwayland-dev \
+            nodejs \
+            libmujs-dev \
+            liblcms2-dev \
+            libarchive-dev \
+            libbluray-dev \
+            libavdevice-dev \
+            cmake \
+            libuchardet-dev \
+            libjack-dev \
+            libopenal-dev \
+            libpipewire-0.3-dev \
+            libpulse-dev \
+            libsndio-dev \
+            libcaca-dev \
+            libdrm-dev \
+            libsixel-dev \
+            wayland-protocols \
+            libxkbcommon-dev \
+            libxpresent-dev \
+            libegl1-mesa-dev \
+            libvulkan-dev \
+            libva-dev \
+            libshaderc-dev \
+            && sudo apt-get clean
+
+    - name: Build mpv binary
+      run: |
+        ./rebuild -j4
+
+    - name: Install mpv binary
+      run: |
+        sudo ./install
+
+    - name: Verify mpv binary
+      run: |
+        mpv --version
+
+    - name: Create debian package
+      run: |
+        # enable source repositories
+        sudo sed -i 's/^Types: deb/Types: deb deb-src/g' /etc/apt/sources.list.d/ubuntu.sources
+        sudo apt-get update
+        # install dependencies mentioned in debian/control
```

Comment:
is there no way to say "take this control file and install everything mentioned there"? a bit unfortunate to duplicate the list here

Reply:
Yes. You can do something like:
```sh
sudo apt-get install -y build-essential devscripts equivs
mk-build-deps -s sudo -i
```
You can also remove the `apt-get build-dep` if you do that.

Reply:
@wsldankers I am currently testing the following command on my local system.

```
equivs-build --full debian/control
```

Which approach is better you think?

Reply:
That does something else, I think. Do you have an objection to the original suggestion?

Reply:
No, I don't have an objection and probably you are right.  After running that command I get `mpv_1.0_amd64.deb` file and I thought it was the right one.